examples/quickstart/jupyter-notebooks/sql-tutorial.ipynb

```diff
@@ -0,0 +1,765 @@
+    "\n",
+    "When `useDefaultValueForNull` is set to `true` (default behavior), Druid stores NULL values as `0` for numeric columns and as `''` for string columns."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "29c24856",
+   "metadata": {
+    "deletable": true,
+    "tags": []
+   },
+   "source": [
+    "## SELECT statement syntax\n",
+    "\n",
+    "Druid SQL supports SELECT statements with the following structure:\n",
+    "\n",
+    "``` mysql\n",
+    "[ EXPLAIN PLAN FOR ]\n",
+    "[ WITH tableName [ ( column1, column2, ... ) ] AS ( query ) ]\n",
+    "SELECT [ ALL | DISTINCT ] { * | exprs }\n",
+    "FROM { <table> | (<subquery>) | <o1> [ INNER | LEFT ] JOIN <o2> ON condition }\n",
+    "[ WHERE expr ]\n",
+    "[ GROUP BY [ exprs | GROUPING SETS ( (exprs), ... ) | ROLLUP (exprs) | CUBE (exprs) ] ]\n",
+    "[ HAVING expr ]\n",
+    "[ ORDER BY expr [ ASC | DESC ], expr [ ASC | DESC ], ... ]\n",
+    "[ LIMIT limit ]\n",
+    "[ OFFSET offset ]\n",
+    "[ UNION ALL <another query> ]\n",
+    "```\n",
+    "\n",
+    "As a general rule, use the LIMIT clause with `SELECT *` to limit the number of rows returned. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8cf212e5-fb3f-4206-acdd-46ef1da327ab",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "## WHERE clause\n",
+    "\n",
+    "Druid SQL uses the [SQL WHERE clause](https://druid.apache.org/docs/latest/querying/sql.html#where) of a SELECT statement to fetch data based on a particular condition.\n",
+    "\n",
+    "In most cases, specifying a time range when using the WHERE clause improves query performance.\n",
+    "This is because Druid partitions data by time and having a time range allows Druid to skip over unrelated data.\n",
+    "\n",
+    "Druid supports range filtering on columns that contain long millisecond values, with the boundaries specified as ISO 8601 time intervals. This is suitable for the `__time` column, long metric columns, and dimensions with values that can be parsed as long milliseconds.\n",
+    "    \n",
+    "For example, the following cell uses a comparison operator on the `__time` field to filter results from a certain time range. The cell limits the results to seven rows for display purposes."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ffca2962-da31-4b9c-adbc-882e35386916",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "endpoint = \"/druid/v2/sql\"\n",
+    "print(f\"\\033[1mQuery endpoint\\033[0m: {druid_host+endpoint}\")\n",
+    "http_method = \"POST\"\n",
+    "\n",
+    "payload = json.dumps({\n",
+    "  \"query\":\"SELECT * FROM \\\"wikipedia-sql-tutorial\\\" WHERE __time >= TIMESTAMP '2015-09-12 23:33:55' LIMIT 7\"\n",
```

Comment:
I'd like to avoid `SELECT *` if possible. If you use fewer columns the output is easier to read. Also selecting fewer columns is more performant with Druid since it only needs to fetch the columns you specify. Consider paring these down.

Reply:
Updated code examples per your suggestion.